documentation/manual/src/doc/index.md

```diff
@@ -1,33 +1,42 @@
 endpoints
 =========
 
-*endpoints* is a Scala library for defining communication protocols over HTTP between
-applications.
-
-Noteworthy features:
-
-- endpoints descriptions are **first-class Scala values**, which can be reused,
-  combined and abstracted over ;
-- a *same* endpoint description can be interpreted in *multiple* ways:
-    - a **client** implementation (JVM and/or Scala.js),
-    - a **server** implementation,
-    - an **OpenAPI** definition file,
-    - thus, your client and documentation are always **consistent** with the actual
-      implementation of your server,
-- **type safety**: you get a compile-time error if you invoke an endpoint but supply incorrect data
-  (requests are guaranteed to be well constructed),
-- **extensibility**: you can introduce both
-  - new *descriptions* that are specific to your application (e.g. the usage
-    of a particular HTTP header),
-  - new *interpreters* for endpoint descriptions (e.g. generation of a RAML documentation),
-- **vanilla Scala**: being written in pure Scala (no macros), the implementation is easy to
-  reason about and IDE friendly.
+*endpoints* is a Scala library for remote communication. It improves the developers productivity by ensuring
+that HTTP servers, HTTP clients, and documentation always agree on the same protocol.
+
+- **maintenance is simplified**: the API documentation is automatically updated when an endpoint is modified ;
+- errors are raised at **compile-time** if endpoints are invoked with incompatible parameters.
+
+Servers, clients and documentation are *all* derived from a *single* source of truth describing the underlying
+protocol details (e.g., which verb, path, query parameters, headers, authentication strategy, etc. to
+use). From the client perspective, calling an HTTP endpoint is as simple as
```

Comment:
you could maybe put a code example to show how an endpoint is defined, with a click bait title: `One definition to rule them (doc, server, client) all`.

When I land on a project, I like to quickly see code in the `README` rather than having to click on multiple link.

Reply:
Good idea, I’ve pushed another commit for that. WDYT?